src/main/java/in/org/projecteka/hiu/ClientError.java

```diff
@@ -41,6 +41,14 @@ public static ClientError consentArtefactNotFound() {
                 new ErrorRepresentation(new Error(CONSENT_ARTEFACT_NOT_FOUND, "Cannot find the consent artefact")));
     }
 
+    public static ClientError consentArtefactGone() {
+        return new ClientError(
+                HttpStatus.GONE,
+                new ErrorRepresentation(new Error(
+                        ErrorCode.CONSENT_ARTEFACT_NOT_FOUND,
+                        "Consent artefact expired")));
+    }
```

Comment:
format it properly, follow the same pattern other in the same class.

Reply:
done